src/lib/components/editor/AiTranslateToolbarButton.svelte

```diff
@@ -200,7 +208,9 @@
 
             const differenceInMinutes = differenceInMilliseconds / (1000 * 60);
 
-            return differenceInMinutes <= 60;
+            //this needs to be changed back after qa testing.
+            //return differenceInMinutes <= 60;
+            return differenceInMinutes <= 5;
```

Comment:
I would prefer that we not put in QA values, since this goes into our master branch. An hour is not that long that QA can't run a test against the real values.